contracts/BaseRouter.sol

```diff
@@ -78,7 +78,8 @@ enum ActionType {
     openPerpetual,
     addToPerpetual,
     veANGLEDeposit,
-    claimRewardsWithPerps
+    claimRewardsWithPerps,
+    addCollateral
```

Comment:
make it explicit that it is linked to borrow module `addCollateralBorrower` or so